### ISSUE TITLE
Can you create static playlists?

### ISSUE THREAD

**Issue by rvzaytsev** [Krakor92/some-custom-dataviews#4]
Let's say I want to create some fixed playlist with a given set of filles. Can I do that?
E.g. I want to just write the songs in my playlist as 
audio:: A.mp3
audio B.mp3
audio C.mp3

and then play those songs in the given order. Is that possible?

**Comment by Krakor92** [comment]
Hi!

I didn't expect to see any traction here as I stopped developing this view a while ago. I'm still using it on a daily basis (the more polished version on the `js-engine` branch though) but I've stopped cramming more features in because the current ones suit me right now.

However, I'm afraid what you're asking isn't possible at the moment. To be honest, I'm a bit surprised that I didn't implement this feature at some point.

Unfortunately, I can't offer you an alternative. I haven't kept up with recent music plugins (if there was any) and the older ones I've referenced don't seem to offer this functionality either.
